lib/app.jsx

```diff
@@ -114,7 +114,12 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 
 	onAppCommand: function( command ) {
 		if ( command != null && typeof command === 'object' && command.action != null && this.props.actions.hasOwnProperty( command.action ) ) {
-			this.props.actions[ command.action ]( command );
+			// Special case for newNote since it needs the note bucket prop from this class
```

Comment:
I'm confused about the comment here. Is the reason we have the special case because we need to preserve the `this` context by calling `onNewNote()` as a method on `this`?